src/source/native.rs

```diff
@@ -1,14 +1,19 @@
+extern crate mio;
```

Comment:
Marker to remove redundant extern create.

Reply:
Fixed.